src/components/missions/sections/index.js

```diff
@@ -1,33 +1,74 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import glamorous from 'glamorous';
 import uniqid from 'uniqid';
-import { Card, Slider } from '@blueprintjs/core';
+import cx from 'classnames';
+import { Classes, Slider, Button } from '@blueprintjs/core';
 import 'normalize.css/normalize.css';
 import '@blueprintjs/core/dist/blueprint.css';
+import logger from '../../logger';
 
 const Worker = ({ workerId }) => <li>Id: {workerId}</li>;
 
 Worker.propTypes = {
   workerId: PropTypes.string,
 };
 
-const CellMission = glamorous(Card)({
+const CellMission = glamorous.div({
   width: 300,
   margin: 10,
+  // }, ({ isSelected, isHovered }) => ({
+  // backgroundColor: isHovered ? 'lightcoral' : 'white',
+  // color: isSelected ? 'red' : 'black',
 });
 
-const Mission = ({ name, clientId, partnerId, managerId, addenda }) => (
-  <CellMission interactive="true" elevation={Card.ELEVATION_TWO}>
-    <b>{name}</b>
-    <p>
-      clientId: {clientId}, partnerId: {partnerId}, managerId: {managerId}
-    </p>
-    <ul>{addenda.map(worker => <Worker key={uniqid()} {...worker} />)}</ul>
-  </CellMission>
-);
+const DisplayButton = glamorous.div({}, ({ isHovered }) => ({
+  display: isHovered ? 'block' : 'none',
```

Comment:
don't always render a Component and play with style to see it or not,
render it only when you want to see it.